src/test/java/core/basesyntax/RegistrationServiceImplTests.java

```diff
@@ -0,0 +1,130 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.IncorrectInputDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceImplTests {
+    private RegistrationServiceImpl service;
+    private final StorageDaoImpl storageDao = new StorageDaoImpl();
+
+    @BeforeEach
+    void setUp() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_ageNullUser_notOk() {
+        User actual = new User();
+        actual.setAge(null);
+        actual.setId(1L);
```

Comment:
Please, remove setId from all methods

Reply:
Please, be attentive in the future, the logic of setting ID implementing in other method, not in the register, so you don't need to set ID here.